src/natlinkcore/natlinkstatus.py

```diff
@@ -500,12 +503,12 @@ def getDragonflyDirectory(self):
         if self.DragonflyDirectory is not None:
             return self.DragonflyDirectory
         try:
-            import dragonfly
+            import dragonfly2
```

Comment:
Hi guys.  This is an error, just FYI. Dragonfly is *always* imported with `import dragonfly`.  dragonfly2 is one of the two *package names*, used only when installing via pip.  Sorry for the confusion!